wallet/src/Pos/Wallet/Web/Methods/Misc.hs

```diff
@@ -14,16 +14,28 @@ module Pos.Wallet.Web.Methods.Misc
        , syncProgress
 
        , testResetAll
+
+       , swaggerSpec
        ) where
 
 import           Universum
 
+import           Control.Lens               ((?~))
+import           Data.Swagger               (Swagger, description, info,
+                                             title, version)
+
+import           Data.Version               (showVersion)
+import qualified Paths_cardano_sl           as CSL
+
+import           Pos.Wallet.Web.Swagger     (toCustomSwagger)
```

Comment:
We're following a rule when imports are grouped like
* Universum
* Modules from standard libraries
* Current project modules